procrastinate/jobs.py

```diff
@@ -22,6 +23,20 @@
 cached_property = getattr(functools, "cached_property", property)
 
 
+class JobInserted(TypedDict):
+    type: Literal["job_inserted"]
+    task_name: str
+    status: str
```

Comment:
> If a job is inserted, isn't its status necessarily todo ?

You're right.

I designed this notification to cover not only inserted job rows with `todo` but any other status. 
In retrospect, I can't think of any other use case and if that happens in the future, we'll deal with it when it comes.

I added the `task_name` for logging purposes at the time but didn't end up using it. 

I will change it to contain the `job_id` since this in itself should be enough for fetching additional job information.
The reason I am mentioning it is that I think it would be a useful (future) feature to expose notifications to end users too.